docs/sphinx/source/advanced_topics.rst

```diff
@@ -198,6 +198,35 @@ levels to use for each chunk.
 
 .. _lev-selection-regrid:
 
+Polytope access to Destination Earth data
+-----------------------------------------
+
+It is possible to access ClimateDT data available on the 'Databridge' for the DestinE ClimateDT also remotely, from other machines,
+using the 'polytope' access. to this end you will need to specify ``engine="polytope"`` when instantiating the `Reader` or permanently, adding
+the argument ``engine: polytope`` as an additional argument in the intake catalog source entry in the corresponding yaml file, under `args:`.
+
+.. code-block:: python
+
+    reader = Reader(model="IFS-NEMO", exp="ssp370", source="hourly-hpz7-atm2d", engine="polytope")
+    data = reader.retrieve(var='2t')
+
+This allows accessing ClimateDT data on the Databridge also remotely from other machines.
+
+In order for this to work you will need to store an access token in the file ``~/.polytopeapirc`` in your home directory.
+Please follow the instructions in the `Polytope documentation <https://github.com/destination-earth-digital-twins/polytope-examples>`_ to 
+set up appropriate credential and create this file. 
+A sample file will look like this:
+
+.. code-block:: text
+
```

Comment:
just tested this, and it seem that adding the email breaks the login. For me it worked only with the user key

Reply:
I tried: if I remove the email from`.polytopeapirc` the this breaks login for me.
My guess is that maybe you are not providing the right email for the login?
Possibly without email specified it guesses it somehow and maybe in your case it gets it right for some reason ?
Or gets it from some environment variable?
I will PM you sending my own credentials, so you can try if using mine specifying email and token works

Reply:
It turns out that the email and token which you need to use are the same which can be found also in your ECMWF account pages. Login to www.ecmwf.int and then go to [https://api.ecmwf.int/v1/key/](https://api.ecmwf.int/v1/key/)

Are these email and token which you were using?

Reply:
I added the ultimate description in the docs for this now